docs/src/prototypes/popups/stickerPicker.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react'
+import {
+  Button,
+  Popup,
+  Menu,
+  dialogBehavior,
+  tabBehavior,
+  tabListBehavior,
+} from '@stardust-ui/react'
+import * as _ from 'lodash'
+import { arrayOfStickerImagesNames, getItemsData } from './helper'
+import GridImagePicker from './GridImagePicker/GridImagePicker'
+
+const tabListItemsContent = [
+  'Popular',
+  'Office drama',
+  'Meme',
+  'Designers',
+  'Dev',
+  'Legal',
+  'Team squatch',
+]
+
+class StickerPicker extends React.Component {
+  state = { activeMenuIndex: 0 }
+  gridPickerRef = React.createRef<GridImagePicker>()
+
+  getStickersData = () => {
+    const index = arrayOfStickerImagesNames[this.state.activeMenuIndex]
+      ? this.state.activeMenuIndex
+      : 0
```

Comment:
The state is initialized with the value of 0, so no need for adding this check.

Reply:
agree